src/test/java/com/a2/backend/service/impl/ReviewServiceImplTest.java

```diff
@@ -0,0 +1,55 @@
+package com.a2.backend.service.impl;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.a2.backend.exception.InvalidUserException;
+import com.a2.backend.model.ReviewCreateDTO;
+import com.a2.backend.repository.UserRepository;
+import com.a2.backend.service.ReviewService;
+import java.util.UUID;
+import lombok.val;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.test.context.support.WithMockUser;
+
+class ReviewServiceImplTest extends AbstractServiceTest {
```

Comment:
We should rename this to something like `ReviewSerciveActiveTest` since it uses DemoRunner